tests/cypress/support/commands.js

```diff
@@ -498,6 +504,9 @@ Cypress.Commands.add('createPolyline', (createPolylineParams) => {
     createPolylineParams.pointsMap.forEach((element) => {
         cy.get('.cvat-canvas-container').click(element.x, element.y);
     });
+    if (createPolylineParams.useDoneButton) {
+        cy.contains('span', 'Done').click();
+    }
```

Comment:
better to use else with the following condition, because both useDoneButton, complete cannot be used together

Reply:
Reworked.